release-notes/docs/3.10.4.md

```diff
@@ -0,0 +1,22 @@
+# 3.10.4 Release Notes
```

Comment:
do we need to add these in: https://github.com/medic/cht-core/blob/master/release-notes/README.md ?

Reply:
yup 🤦🏻